src/store/getters/state-getters.ts

```diff
@@ -25,6 +24,31 @@ import { getMaxStep } from './resource-getters'
 // The getters in this file return values from the current state model.
 //
 
+/** Whether the current filing is an Incorporation. */
+export const isIncorporationFiling = (state: StateIF): boolean => {
+  return getFilingType(state) === FilingTypes.INCORPORATION_APPLICATION
+}
+
+/** Whether the current filing is a Dissolution. */
+export const isDissolutionFiling = (state: StateIF): boolean => {
+  return getFilingType(state) === FilingTypes.DISSOLUTION
+}
+
+/** The current filing type. */
+export const getFilingType = (state: StateIF): FilingTypes => {
+  return state.stateModel.tombstone.filingType
+}
+
+/** The current filing name. */
+export const getFilingName = (state: StateIF): FilingNames => {
+  switch (getFilingType(state)) {
+    case FilingTypes.INCORPORATION_APPLICATION:
+      return FilingNames.INCORPORATION_APPLICATION
+    case FilingTypes.DISSOLUTION:
+      return FilingNames.DISSOLUTION_FILING
+  }
+}
```

Comment:
Add a default case in the switch or a fallback return here.

I guess this didn't cause a lint error because it's smart enough to know there are only 2 cases, and you handle them both. Still, the code can be more robust here.

Reply:
K, will add a default / fall back to the enum, otherwise we will have to modify the return type. Unless the latter is more appealing, thoughts?

Reply:
Don't change the enum. Just return null (which is, handily, falsy).

Reply:
oh right on!

Reply:
Perfect.